src/components/content/content.test.js

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import Content from "./content.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import {
+  contentTitle,
+  contentBody,
+  contentPreview,
+} from "../../../cypress/locators/content/index";
+
+context("Tests for Content component", () => {
+  describe("should check Content component properties", () => {
+    it.each(["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"])(
+      "should check %s as children for Content component",
+      (children) => {
+        CypressMountWithProviders(<Content title="Title">{children}</Content>);
+        contentBody().should("have.text", children);
+      }
+    );
+
+    it.each([
+      ["mp150ú¿¡üßä", "primary"],
+      ["!@#$%^*()_+-=~[];:.,?{}&\"'<>", "secondary"],
+    ])("should check %s as title for Content component", (title, variant) => {
+      CypressMountWithProviders(
+        <Content title={title} variant={variant}>
+          This is an example of some content
+        </Content>
+      );
+      contentTitle().should("have.text", title);
+    });
+
+    it("should check inline parameter is enabled for Content component", () => {
+      CypressMountWithProviders(
+        <Content title="Title" inline>
+          This is an example of some content
```

Comment:
to avoid providing in each test this children we can build the custom component with will render it for us, and just override needed value